generate_stubs.py

```diff
@@ -48,7 +52,7 @@
     with tempfile.NamedTemporaryFile() as config:
         config.write('[mypy]\nignore_errors = True\n'.encode())
         config.flush()
-        subprocess.check_call([sys.executable, '-m' 'mypy', f'{DIRECTORY}/{MODULE_NAME}', f'--config-file={config.name}'])
+        subprocess.check_call([sys.executable, '-m' 'mypy', f'{DIRECTORY}/{MODULE_NAME}', f'--config-file={config.name}'], env=env)
```

Comment:
Let's add  '--pretty' too for a nice error print, if it's the case.
Tested locally.
Can open a separate PR if needed though.

Reply:
You mean to not use the config file?
Feel free to push to same PR IMO. Just resolved the Windows issue

Reply:
Isn't config file just for setting `ignore_errors = True` ? Could be inline CLI option too.
`--pretty` is for printing the errors nicely, otherwise the source of error is not printed/clear in my experience.

Reply:
Yup, just to set that flag. 

> Could be inline CLI option too.

To `mypy`? Not sure, didn't found it on first glance. Its however possible to `from mypy import api` but I've took this route instead

Regardging `--pretty`, by default it is if ran from terminal (script directly). At build time via CMake route, if it goes through Ninja it won't be colorized. Are you using Ninja? In that case, `--pretty` would force colorized output I assume?